src/main/java/com/microsoft/sqlserver/jdbc/SQLServerInfoMessage.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+
+package com.microsoft.sqlserver.jdbc;
+
+import java.sql.SQLException;
+
+public final class SQLServerInfoMessage extends StreamPacket implements ISQLServerMessage {

```

Comment:
javadoc for class description

Reply:
done

Reply:
I also added javadoc class description for SQLServerWarning